<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BachelorarbeitClass</class>
 <widget class="QMainWindow" name="BachelorarbeitClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bachelorarbeit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="oglContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbDisplay">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QRadioButton" name="rbDensity">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>29</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Density</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbVelocity">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>49</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbIntegration">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Integration</string>
    </property>
    <widget class="QRadioButton" name="rbEuler">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>29</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Euler</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbRK2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>49</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Runge Kutta 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbSplatRadius">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Splat Radius</string>
    </property>
    <widget class="QSlider" name="hsDensityRadius">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Density</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsVelocityRadius">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblDensityRadius">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVelocityRadius">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbDissipation">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Dissipation</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Density</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsVelocityDissipation">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="hsDensityDissipation">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDensityDissipation">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>31</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.99</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVelocityDissipation">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.99</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbReset">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbColor">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>500</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbProjection">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Projection</string>
    </property>
    <widget class="QRadioButton" name="rbJacobi">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>29</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Jacobi</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbGauss">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>49</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gauss-Seidel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbNoProject">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>71</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No Projection</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Iterations:</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsIterations">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblIterations">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>94</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbPause">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbVelocity">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Velocity</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplicator:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMultiplicator">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>23</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
    <widget class="QSlider" name="hsMultiplicator">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Bachelorarbeit.qrc"/>
 </resources>
 <connections/>
</ui>
